target/linux/ath79/dts/ar7161_trendnet_tew-673gru.dts

```diff
@@ -97,13 +98,19 @@
 	wifi@11,0 {
 		compatible = "pci168c,0029";
 		reg = <0x8800 0 0 0 0>;
-		qca,no-eeprom;
+		nvmem-cells = <&macaddr_lan 0>, <&cal_caldata_1000>;
+		nvmem-cell-names = "mac-address", "calibration";
+		#gpio-cells = <2>;
+		gpio-controller;
```

Comment:
If we don't use the ath9k chip's gpio, I think these gpio properties are useless.